dotnet/src/webdriver/BiDi/Modules/Script/LocalValue.cs

```diff
@@ -23,21 +23,21 @@
 namespace OpenQA.Selenium.BiDi.Modules.Script;
 
 [JsonPolymorphic(TypeDiscriminatorPropertyName = "type")]
-[JsonDerivedType(typeof(Number), "number")]
-[JsonDerivedType(typeof(String), "string")]
-[JsonDerivedType(typeof(Null), "null")]
-[JsonDerivedType(typeof(Undefined), "undefined")]
-[JsonDerivedType(typeof(Channel), "channel")]
-[JsonDerivedType(typeof(Array), "array")]
-[JsonDerivedType(typeof(Date), "date")]
-[JsonDerivedType(typeof(Map), "map")]
-[JsonDerivedType(typeof(Object), "object")]
-[JsonDerivedType(typeof(RegExp), "regexp")]
-[JsonDerivedType(typeof(Set), "set")]
+[JsonDerivedType(typeof(NumberLocalValue), "number")]
+[JsonDerivedType(typeof(StringLocalValue), "string")]
+[JsonDerivedType(typeof(NullLocalValue), "null")]
+[JsonDerivedType(typeof(UndefinedLocalValue), "undefined")]
+[JsonDerivedType(typeof(ChannelLocalValue), "channel")]
+[JsonDerivedType(typeof(ArrayLocalValue), "array")]
+[JsonDerivedType(typeof(DateLocalValue), "date")]
+[JsonDerivedType(typeof(MapLocalValue), "map")]
+[JsonDerivedType(typeof(ObjectLocalValue), "object")]
+[JsonDerivedType(typeof(RegExpLocalValue), "regexp")]
+[JsonDerivedType(typeof(SetLocalValue), "set")]
 public abstract record LocalValue
 {
-    public static implicit operator LocalValue(int value) { return new Number(value); }
-    public static implicit operator LocalValue(string value) { return new String(value); }
+    public static implicit operator LocalValue(int value) { return new NumberLocalValue(value); }
+    public static implicit operator LocalValue(string value) { return new StringLocalValue(value); }
```

Comment:
Who knows what happens if we have `StringLocalValue(null)`

```suggestion
    public static implicit operator LocalValue(string? value) { return value is null ? new NullLocalValue() : new StringLocalValue(value); }
```